Kaguya/Kaguya/Discord/Attributes/CommandAttributes.cs

```diff
@@ -51,6 +52,21 @@ internal class ModuleAttribute : Attribute
 		public ModuleAttribute(CommandModule module) { Module = module; }
 	}
 
+	[AttributeUsage(AttributeTargets.Method)]
+	internal class ExamplesAttribute : Attribute
```

Comment:
Examples will look cleaner in code if it's one per attribute instead of several with \n separators